src/main/resources/db/migration/V7__imrt_ddl.sql

```diff
@@ -0,0 +1,13 @@
+/******************************************************************************
+* Add organization ingest fields to the item table:
+*
+*   organization_type_id
+*   organization_name
+******************************************************************************/
+ALTER TABLE item ADD COLUMN organization_type_id VARCHAR NOT NULL DEFAULT '';
+ALTER TABLE item ADD COLUMN organization_name VARCHAR NOT NULL DEFAULT '';
+
```

Comment:
@millerts You mentioned we had lengths for the various fields now - are there limits for these ones that we could add to the schema? Also should we have a Jira for going back and setting the lengths on all the other fields?

Reply:
I will create a JIRA for both.  After you asked this question I realized that we don't have this in the IDD and I need to verify widths with the client.